### PR TITLE
install instruction updates, harmonize w/ tools page

### PR DESCRIPTION
**Pull request by mitzimorris**
Edits to the "Getting Started" and "Toolkit" pages:

- better use of tables to present lists of software
- links to introductory tutorials
- tweaks to theming

### CODE REVIEW

File: tools/tools.qmd [hunk 1]
```diff
@@ -1,187 +1,93 @@
...
+| **Visual Studio Code** | [**Stan VSCode Extension**](https://marketplace.visualstudio.com/items?itemName=stan-dev.stan-vscode) | Stan language support with syntax highlighting, autocompletion, and snippets. |
+| **Emacs**              | [**stan-mode**](https://github.com/stan-dev/stan-mode) | Major mode for Stan with syntax highlighting and indentation. |
+| **Vim**                | [**stan-vim**](https://github.com/eigenfoo/stan-vim) | Syntax highlighting, indentation, and code folding for Stan in Vim. |
+| **Atom**               | [**language-stan**](https://github.com/stan-dev/atom-language-stan) | Stan language support in Atom editor. |
+| **Sublime Text**       | [**SublimeStan**](https://github.com/djsutherland/sublime-stan) | Syntax highlighting for Stan in Sublime Text. |
+| **JupyterLab**         | [**jupyterlab-stan-highlight**](https://www.npmjs.com/package/jupyterlab-stan-highlight) | Syntax highlighting for Stan code blocks in JupyterLab. |
+| **Markdown Editors**   | **Pandoc**, **Pygments** | Stan syntax highlighting in markdown via [Pandoc](https://pandoc.org/) and [Pygments](https://pygments.org/). |
```
Comment (WardBrian) [inline tools/tools.qmd]: I think the links should be in the second column for consistency. It also is probably worth linking to where the support is implemented, like on the current site

File: tools/tools.qmd [hunk 2]
```diff
@@ -1,187 +1,93 @@
...
+| **LaTeX**              | [**lstbayes**](https://github.com/jrnold/lstbayes) | LaTeX listings for Stan syntax highlighting. |
+
```
Comment (WardBrian) [inline tools/tools.qmd]: This table dropped the mentions of Highlight.js and Prism from the current site -- intentional?

File: install/install.qmd [hunk 3]
```diff
@@ -283,27 +284,71 @@ Common Points of failure; how to address them.
...
-* CmdStanPy User's Guide Examples
+## High-level Stan Interfaces
 
-* "Getting Started with Bayesian Statistics using Stan and Python" by Bob Carpenter
+| **Language** | **Tool** | **Description** |
+|--------------|----------|-----------------|
+| **R**        | [**BRMS**](https://paul-buerkner.github.io/brms/){target="_blank"} | Use extended `lme4`-like formula syntax to specify and fit multivariate and multilevel models in Stan. *(Requires CmdStanR and C++ compiler.)* |
+| **R**        | [**RStanArm**](https://mc-stan.org/rstanarm){target="_blank"} | Provides stable, efficient Stan versions of R model-fitting packages. *(Stan models are pre-compiled, no C++ compiler needed.)* |
+| **R**        | [**Rethinking**](https://github.com/rmcelreath/rethinking){target="_blank"} | Accompanies the book and course materials for [Statistical Rethinking, 2nd Ed](https://www.taylorfrancis.com/books/mono/10.1201/9780429029608/statistical-rethinking-richard-mcelreath){target="_blank"} by Richard McElreath.|
+| **Python**   | [**Bambi**](https://bambinos.github.io/bambi/){target="_blank"} | Bayesian models in Python using formula syntax, built on top of Stan and PyMC. |
```
Comment (WardBrian) [inline install/install.qmd]: Does bambi really have the option to use Stan? I can't find it in their docs

File: install/install.qmd [hunk 4]
```diff
@@ -283,27 +284,71 @@ Common Points of failure; how to address them.
...
-### General Overview
+## Introductory Notebooks, Vignettes, and Tutorials
 
-* StanCon 2023 "Fundamentals of Stan"
 
-* Course notes: *Software for MCMC* from Odd Kolbjørnsen, Spring 2023, Oslo Uni\
-\fontsize{9pt}{9.4}\selectfont
-www.uio.no/studier/emner/matnat/math/STK4051/v23/timeplan/lecture_12_softwareformcmc.pdf
-\normalsize
+<table>
```
Comment (WardBrian) [inline install/install.qmd]: Is there not a way to write this using the normal markdown table syntax? I think the HTML one is hard to read, and appreciated how the rest of these changes removed some

File: install/install.qmd [hunk 5]
```diff
@@ -283,27 +284,71 @@ Common Points of failure; how to address them.
 
 See CmdStan Guide section [Troubleshooting the Installation](https://mc-stan.org/docs/cmdstan-guide/installation.html#troubleshooting-the-installation){target="_blank"} for further details.
 
-## Introductory Notebooks, Vignettes, and Tutorials
-
-### Python
-
-* CmdStanPy User's Guide Examples
+## High-level Stan Interfaces
```
Comment (WardBrian) [inline install/install.qmd:284]: I like the idea of this, but it's probably worth noting somewhere in the table which of the above interfaces you need to install to use these (I think all 3 of the R options here can use _either_ RStan or CmdStanR)

File: tools/tools.qmd [hunk 6]
```diff
@@ -1,187 +1,93 @@
...
-</table>
+# Stan Toolkit
+
+A curated collection of tools and interfaces to help you work effectively with Stan across various programming environments and stages of your modeling workflow.
+
+---
+
+## Language-Specific Stan Interfaces
+
+*Write, compile, and run Stan models directly within your programming environment.*
```
Comment (WardBrian) [inline tools/tools.qmd:13]: Feels maybe a bit repetitive with the install page, but that's not necessarily a bad thing. We should probably link to the install matrix still

File: tools/tools.qmd [hunk 7]
```diff
@@ -1,187 +1,93 @@
...
+
+## High-Level Modeling Interfaces
+
+*Simplify model specification using familiar syntax and frameworks.*
+
+| **Language** | **Tool** | **Description** |
+|--------------|----------|-----------------|
+| **R**        | [**brms**](https://paul-buerkner.github.io/brms/) | Use extended `lme4`-like formula syntax to specify and fit multivariate and multilevel models in Stan. *(Requires CmdStanR and C++ compiler.)* |
+| **R**        | [**rstanarm**](https://mc-stan.org/rstanarm) | Provides stable, efficient Stan versions of R model-fitting packages. *(Stan models are pre-compiled, no C++ compiler needed.)* |
+| **Python**   | [**Bambi**](https://bambinos.github.io/bambi/) | Bayesian models in Python using formula syntax, built on top of Stan and PyMC. |
```
Comment (WardBrian) [inline tools/tools.qmd]: Same question re-bambi

File: tools/tools.qmd [hunk 8]
```diff
@@ -1,187 +1,93 @@
...
+| **Sublime Text**       | [**SublimeStan**](https://github.com/djsutherland/sublime-stan) | Syntax highlighting for Stan in Sublime Text. |
+| **JupyterLab**         | [**jupyterlab-stan-highlight**](https://www.npmjs.com/package/jupyterlab-stan-highlight) | Syntax highlighting for Stan code blocks in JupyterLab. |
+| **Markdown Editors**   | **Pandoc**, **Pygments** | Stan syntax highlighting in markdown via [Pandoc](https://pandoc.org/) and [Pygments](https://pygments.org/). |
+| **LaTeX**              | [**lstbayes**](https://github.com/jrnold/lstbayes) | LaTeX listings for Stan syntax highlighting. |
+
+---
+
+## Additional Notes
+
+- **Recommendation for New Users**: We suggest starting with **CmdStanR** (for R users) or **CmdStanPy** (for Python users) along with high-level interfaces like **brms** or **rstanarm** for ease of use.
```
Comment (WardBrian) [inline tools/tools.qmd]: Redundant with above recommendations in-line in the table, I think

**Review by WardBrian — COMMENTED** [review]
Looks good! A few scattered questions

File: tools/tools.qmd (hunk 2 above)
Comment (mitzimorris) [reply tools/tools.qmd]: no - will restore

File: tools/tools.qmd (hunk 1 above)
Comment (mitzimorris) [reply tools/tools.qmd]: good point -  added source code links

File: install/install.qmd (hunk 3 above)
Comment (mitzimorris) [reply install/install.qmd]: removed.

File: install/install.qmd (hunk 4 above)
Comment (mitzimorris) [reply install/install.qmd]: yes, changed to markdown.

File: tools/tools.qmd (hunk 7 above)
Comment (mitzimorris) [reply tools/tools.qmd]: removed mention of Bambi.

File: tools/tools.qmd (hunk 8 above)
Comment (mitzimorris) [reply tools/tools.qmd]: removed.

File: install/install.qmd (hunk 5 above)
Comment (mitzimorris) [reply install/install.qmd:284]: RStanArm doesn't require Rstan - it does require rstantools - https://cran.r-project.org/web/packages/rstanarm/index.html    that's the selling point for RStanArm - it installs easily from CRAN.

File: tools/tools.qmd (hunk 6 above)
Comment (mitzimorris) [reply tools/tools.qmd:13]: excellent suggestion.   added cross links between pages.

**Comment by mitzimorris** [comment]
ready for re-review

File: tools/tools.qmd [hunk 9]
```diff
@@ -73,21 +76,16 @@ A curated collection of tools and interfaces to help you work effectively with S
...
+| **Sublime Text**       | [**SublimeStan**](https://github.com/djsutherland/sublime-stan){target="_blank"} | Syntax highlighting for Stan in Sublime Text. |
+| **JupyterLab**         | [**jupyterlab-stan-highlight**](https://www.npmjs.com/package/jupyterlab-stan-highlight){target="_blank"} | Syntax highlighting for Stan code blocks in JupyterLab. |
+| **Javascript**         | [**Prism**](https://prismjs.com/){target="_blank"} </br> [**Highlight.js**](https://github.com/highlightjs/highlight.js){target="_blank"} | Lightweight syntax highlighting library - [(Source code)](https://github.com/PrismJS/prism/blob/master/components/prism-stan.js){target="_blank"}. </br> Syntax highlighter written in javascript - [(Source code)](https://github.com/highlightjs/highlight.js/blob/main/src/languages/stan.js){target="_blank"} |
+| **Markdown Editors**   | [**Pandoc**](https://pandoc.org/){target="_blank"} </br> [**Pygments**](https://pygments.org/){target="_blank"} | Stan syntax highlighting in markdown. [(Source code)](https://github.com/pygments/pygments/blob/master/pygments/lexers/modeling.py){target="_blank"} |
```
Comment (WardBrian) [inline tools/tools.qmd]: The upstream source for pandoc is https://invent.kde.org/frameworks/syntax-highlighting/-/blob/master/data/syntax/stan.xml

**Review by WardBrian — APPROVED** [review]

File: tools/tools.qmd (hunk 9 above)
Comment (mitzimorris) [reply tools/tools.qmd]: thanks for this - added.

**Pull request merged**